spec/stored_queries/ontology/ontology_get_associated_samples.yaml

```diff
@@ -45,9 +49,18 @@ query: |
         }
   )
   LET total_count=COUNT(results)
-  LET limited=(
+  LET filtered=(
     FOR r in results
+      FILTER @user_id == r.sample_access.acls.owner 
+        OR @user_id IN r.sample_access.acls.admin
+        OR @user_id IN r.sample_access.acls.read
+        OR r.sample_access_acls.pubread
```

Comment:
the other lines use `r.sample_access.acls` -- is this line correct?

Reply:
Thanks for pointing it out. I have fixed the typo.